tchannel/serializer/raw.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from tchannel.schemes import RAW
+
+
+class RawSerializer(object):
+    def __init__(self):
```

Comment:
The thrift serializer needs arg_type and result_type class in order to do deserializer. json, raw don't need that but in order to keep three look same.  I make three all this way